src/components/Router/index.js

```diff
@@ -204,11 +204,11 @@ class HistoryRouter extends Component {
 
   initialStep = () => this.state.initialStep === this.state.step && this.state.flow === 'captureSteps'
 
-  changeFlowTo = (newFlow, newStep=0) => {
+  changeFlowTo = (newFlow, newStep = 0, addStepToHistory) => {
```

Comment:
We should disable navigation if `addStepToHistory` is `false`. See line 198

Reply:
Why should we disable navigation?

Reply:
Because when I open the crossDevice client I can see the back button and when I click on it, it doesn't take me anywhere. We always disable the back button on the first screen.